<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadingMeaningGroupWidget</class>
 <widget class="QWidget" name="ReadingMeaningGroupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>84</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="onReadingsLabel">
     <property name="text">
      <string>Readings on:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="kunReadingsLabel">
     <property name="text">
      <string>Readings kun:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="englishMeaningLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>English meanings:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="frenchMeaningLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>French meanings:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="onReadingsLayout"/>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="kunReadingsLayout"/>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="englishMeaningsLayout"/>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="frenchMeaningsLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
